test/test_autograd.py

```diff
@@ -3992,8 +3992,7 @@ def test_increment_version(self):
 
         with torch.inference_mode():
             a = torch.rand(5, requires_grad=True)
-        msg = "update to inference tensor outside InferenceMode"
-        with self.assertRaisesRegex(RuntimeError, msg):
+            # does not error
```

Comment:
You need to fix indentation !
This is expected to check outside of inferenc emode !

Reply:
pushed too quickly... fixing the indentation.

> This is expected to check outside of inferenc emode !

wdym? (now that we aren't raising an error when you call this fn on a tensor created in inference mode, I figured I would just make the test assert that we can call `increment_version` on a tensor created in inference mode without blowing up)